tests/ecosystem/deployment/test_ocs_basic_install.py

```diff
@@ -63,10 +63,11 @@ def test_deployment(self):
 
         # Download installer
         installer = get_openshift_installer(
-            DEPLOYMENT['installer_version']
+            DEPLOYMENT['installer_version'],
+            force_download=True,
         )
         # Download client
-        get_openshift_client()
+        get_openshift_client(force_download=True)
```

Comment:
Same here, but as I pointed in some our previous conversation I think that client is RUN specific I think and should be done each run in case the user will change the version in config.

Reply:
In current state of the tests, the client is downloaded only here in the deployment test (no other test is using the `get_openshift_client()` function), so I think it is ok, to force the download in the same way as for the installer... It might change once we will come with some more flexible approach to get the client for each test  (e.g. fixture)...

Reply:
Yep, for sure the download client each time should be implemented in different PR. Just pointed to use that section here as you did. Thanks